src/main/java/mate/academy/spring/service/mapper/UserDtoMapper.java

```diff
@@ -0,0 +1,24 @@
+package mate.academy.spring.service.mapper;
+
+import mate.academy.spring.model.User;
+import mate.academy.spring.model.dto.UserRequestDto;
+import mate.academy.spring.model.dto.UserResponseDto;
+import org.springframework.stereotype.Component;
+
+@Component
+public class UserDtoMapper {
+    public UserResponseDto parse(User user) {
+        UserResponseDto userResponseDto = new UserResponseDto();
+        userResponseDto.setId(user.getId());
+        userResponseDto.setName(user.getName());
+        userResponseDto.setLastName(user.getLastName());
+        return userResponseDto;
+    }
+
+    public User toUser(UserRequestDto userRequestDto) {
```

Comment:
It's a bad practice to add code that is not needed now to PR, let' remove it for now

Reply:
done